backend/server.js

```diff
@@ -24,6 +24,23 @@ import { GIMessage } from '~/shared/domains/chelonia/GIMessage.js'
 
 const { CONTRACTS_VERSION, GI_VERSION } = process.env
 
+const securityHeaders = {
+  // This is the most likely to break things now; it may need changing when sandboxing or federation is implemented.
+  'Content-Security-Policy': "default-src 'none'; script-src 'unsafe-eval'; script-src-elem 'self'; script-src-attr 'none'; style-src 'self'; style-src-elem 'self'; style-src-attr 'none'; img-src 'self' blob:; font-src 'self'; connect-src 'self'; media-src 'self'; prefetch-src 'self'; worker-src 'self'; frame-ancestors 'none'; form-action 'self'; upgrade-insecure-requests; manifest-src 'self'",
```

Comment:
This will cause issues with the emoji images, since those are loaded externally. See #2334. Either `img-src` will need to be expanded, or the sheet be embedded locally (preferred solution).

Reply:
Is using `img-src 'self' blob: https://unpkg.com` OK for now? Otherwise, installing and redistributing emoji sheets via `dist/assets` seems a bit tricky regarding e.g. licensing issues.

Reply:
In terms of solving the CSP issue, yes (although you need to check manually, it could interfere with Cross-Origin-Embedder-Policy as well).

There preferred (to me) solution would be embedding the sheet. There could be some licensing concerns, but I'm not sure those concerns are alleviated by hotlinking. This is best left to a lawyer or to the copyright holder. (Sidenote: fonts designs _generally_ aren't under copyright in the _US_, but I'm guessing emoji designs likely are, and even if they weren't, there's other places to consider).

Reply:
An alternative solution is to drop the Apple emojis and use an emoji set with a free license. E.g., https://github.com/mozilla/fxemoji